scripts/release/asset_publish.py

```diff
@@ -311,17 +311,29 @@ def _str2bool(v: str) -> bool:
                 env_version = match.group(2)
                 logger.print(f"Env name:version => {env_name}:{env_version}")
                 logger.print(f"Using final_version {final_version} to check if env registered")
+
+                mlclient = MLClient(
+                    DefaultAzureCredential(),
+                    subscription_id=subscription_id,
+                    registry_name=registry_name,
+                )
```

Comment:
Consider moving this outside of the loop.

Reply:
Thank you done